config/rubocop-2.4.yml

```diff
@@ -1342,5 +1361,5 @@ Style/WordArray:
   Description: Use %w or %W for arrays of words.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#percent-w
   Enabled: true
-  MinSize: 0
+  MinSize: 3
```

Comment:
Interesting we already had 0, maybe we should keep it then? I was under the impression this was a new thing

Reply:
My bad, this is also present on the rubocop file targeting ruby 2.1 https://github.com/bespokepost/code-climate-config/blob/master/config/rubocop.yml#L1346 

I will undo.